scripts/start_celo_ganache_ci.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+# Copyright 2020 ChainSafe Systems
+# SPDX-License-Identifier: LGPL-3.0-only
+
+
+# Exit on failure
+set -ex
+
+PORT=${PORT:-8545}
+
+echo "Running celo ganache..."
+node  ./ganache-cli/cli.js -q --blockTime 1 -p $PORT --account "0x000000000000000000000000000000000000000000000000000000616c696365,100000000000000000000" --account "0x0000000000000000000000000000000000000000000000000000000000626f62,100000000000000000000" --account "0x00000000000000000000000000000000000000000000000000636861726c6965,100000000000000000000"  --account "0x0000000000000000000000000000000000000000000000000000000064617665,100000000000000000000" --account "0x0000000000000000000000000000000000000000000000000000000000657665,100000000000000000000" &
+# Otherwise CI will run tests before ganache has started
+sleep 3
```

Comment:
Is there any reason to use this instead of `SILENT=true ./scripts/start_celo_ganache.sh`?

Reply:
couse you can see that it is also differs by command i use to start it, and since the old one script still can be used in some local staff this one is suppose so use only in CI,

Reply:
Let's update start_celo_ganache.sh to just be used for local testing (remove if statement and just have the `else` case). The `SILENT` env var is only useful for CI, I don't see any reason to start with `&` locally

Reply:
good point